openshift/authorize/secret/namespaceSecret.yml

```
@@ -0,0 +1,36 @@
+apiVersion: v1
+
+
+
+apiVersion: template.openshift.io/v1
+kind: Template
+
+parameters:
+  - description: namespace
+    displayName: namespace
+    name: NAMESPACE
+    required: true
+  - description: airflow_namespace
+    displayName: airflow_namespace
+    name: AIRFLOW_NAMESPACE
+    required: true
+  - description: ggircs_namespace
+    displayName: ggircs_namespace
+    name: GGIRCS_NAMESPACE
+    required: true
+  - description: ciip_namespace
+    displayName: ciip_namespace
+    name: CIIP_NAMESPACE
+    required: true
+
+objects:
+  - kind: Secret
+    apiVersion: v1
+    metadata:
+      name: cas-namespaces
+      namespace: ${NAMESPACE}
```

```suggestion
      namespace: ${NAMESPACE}
      labels:
        created-by: cas-pipeline
```
This will allow us to easily find all of the objects created by the provisioning script